analysis/benchmark_results.py

```diff
@@ -328,7 +328,7 @@ def _bug_coverage_growth_plot(self, filename, logscale=False):
         """Bug coverage growth plot."""
         plot_filename = self._prefix_with_benchmark(filename)
         self._plotter.write_coverage_growth_plot(
-            data_utils.add_bugs_covered_column(self._benchmark_df),
+            self._benchmark_df,
```

Comment:
How does this work keeping the crash_key column as-is ? Wont this have duplicate rows for particular (trial_id, fuzzer, time) combination due to different crash keys.

Reply:
yes this is a still(?) a problem.  I think you currently have those duplicates in the data for every other chart as the duplicates were only filtered for the bug_coverage_growth plot?   However, I think the data (identifiable bugs) could still be useful to keep for unique bug analysis.  Similar to the unique coverage charts that are generated now.  In keeping the duplicate rows with different `crash_key` values, the `edges_covered` and `bugs_covered` numbers are all the same for a unique (benchmark, fuzzer, trial_id, time) combination.

This is probably why you have a really high count (45) for libfuzzer samples on arrow_parquet-arrow-fuzz summary?

Possible fixes:
1. Add a filter to `get_benchmark_snapshot` to drop the `crash_key` column and remove duplicates.
2. Add a filter for both edge and bug coverage growth plots to remove duplicates.

Reply:
Added the filter after the data is exported to csv.